examples/model_load_store_lora/bentofile.yaml

```diff
@@ -0,0 +1,18 @@
+service: "service.py:StableDiffusion"
```

Comment:
let's also add model spec here. I think we didn't document this very well, but it's something like

```
model:
  tag: sd2
```

read https://github.com/bentoml/BentoML/blob/main/src/bentoml/_internal/bento/build_config.py#L716 for reference